### PR TITLE
Set demo app form config from url params

### PR DESCRIPTION
**Pull request by BboyStatix**
## Ticket Link
https://toruslabs.atlassian.net/browse/PD-1004
## What does this PR do?
Enables setting form configs for whitelabel, chain, and authmode based on URL search parameters
Also includes minor UI fixes

### CODE REVIEW

File: examples/vue-app/src/home.vue [hunk 1]
```diff
@@ -328,17 +325,18 @@ export default Vue.extend({
     WhitelabelExample: WhitelabelExample,
     CustomUiContainer,
   },
-  mounted() {
-    const storedConfig = sessionStorage.getItem("web3AuthExampleConfig");
-    const finalStoredConfig = JSON.parse(storedConfig || "{}");
-    this.config = merge(this.config, finalStoredConfig);
-    if (finalStoredConfig.uiMode) this.config.uiMode.whitelabel.loginMethodsOrder = finalStoredConfig.uiMode.whitelabel.loginMethodsOrder;
```
Comment (BboyStatix) [inline examples/vue-app/src/home.vue:335]: @lionellbriones what is the reason for this line?

Comment (BboyStatix) [reply examples/vue-app/src/home.vue:335]: @himanshuchawla009

Comment (BboyStatix) [reply examples/vue-app/src/home.vue:335]: @himanshuchawla009 `finalStoredConfig` is already merged into `this.config` so why do we need do assign the loginMethodsOrder here?

**Review by lionellbriones — APPROVED** [review]
lgtm

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**